website_sale_product_pack/models/product_template.py

```diff
@@ -33,8 +40,54 @@ def check_website_published(self):
             if published:
                 raise ValidationError(
                     _(
-                        "You can't unpublished product (%s) for a published "
-                        "pack parents (%s)"
+                        "You can't unpublished product (%(product_name)s) for a"
+                        "published pack parents (%(pack_parents)s)"
                     )
-                    % (", ".join(rec.name, published.mapped("name")))
+                    % {
+                        "product_name": rec.name,
+                        "pack_parents": ", ".join(published.mapped("name")),
+                    }
                 )
+
+    # Neccessary for the website_sale_product_pack module because the price in /shop
+    # is calculated by the product.template.price_compute method
+    def price_compute(
+        self, price_type, uom=False, currency=False, company=False, date=False
+    ):
```

Comment:
Wouldn't this override belong to the base module? Isn't there a common method there we can use?